bin/render.js

```diff
@@ -47,8 +48,9 @@ module.exports = (type, callback, opt) => {
 
   // instantiate view and invoke headless render method
   function render(vlSpec) {
+    const specTheme = themes[vlSpec.usermeta.embedOptions.theme];
```

Comment:
Should we also support a command line argument to set the theme? 

And if we support Vega-Lite, should we also support Vega?